packages/reakit/src/Grid/Grid.ts

```diff
@@ -1,11 +1,25 @@
-import PropTypes from "prop-types";
+import * as PropTypes from "prop-types";
 import { theme } from "styled-tools";
 import { bool, value } from "../_utils/styledProps";
 import styled from "../styled";
 import as from "../as";
 import Box from "../Box";
 
-const Grid = styled(Box)`
+export interface GridProps {
+  Item: React.ReactType;
```

Comment:
`Item` isn't a component prop. It's a static member. :P

Reply:
Oops :)